DNSExfiltration.md

```diff
@@ -14,6 +14,9 @@ This is a common theme where an attacker gets specific information about where t
 
   <img src="https://raw.githubusercontent.com/step-security/supply-chain-goat/main/images/DNSExfiltration.png" alt="DNS exfiltration" width="800">
 
+## How StepSecurity mitigates threat?
+StepSecurity's tool analyzes the outbound calls made by the workflow and recommends the appropriate policy containing the allowed outbound endpoints. Every other outbound call that is not included in the allowed endpoints is blocked to prevent a potential DNS Exfiltration attack.
```

Comment:
You can remove the word `tool` from this sentence. StepSecurity analyzes...

Instead of `every other outbound call`, you can make it `Any outbound call not in the list of allowed endpoints is blocked to prevent a potential DNS Exfiltration attack`